### DISCUSSION TITLE
self signed cert not allowing connection to copilot servers

### DISCUSSION THREAD

**Discussion by s0r3-glitch** [community/community#7997] [Copilot Conversations] [answered]
I am trying to connect to copilot through my company's VPN which always returns data with a self signed cert. When looking at the copilot output it says that it cant connect due to the self signed cert (keep in mind the self signed cert is expected behavior) is there any way to have copilot ignore the self signed cert or is that something that cant be changed?

**Comment by davecheney** [comment, marked as answer]
Not at the moment I'm afraid.

**Reply by s0r3-glitch** [reply]
Do you know if its a planned feature?
